examples/irccat.json

```diff
@@ -8,7 +8,10 @@
     "tls_key": "",
     "tls_cert": "",
     "listeners": {
-      "generic": true,
+      "generic": {
```

Comment:
I think it would be nice if we could make this backwards-compatible if we can. I haven't touched any Go in ages so I don't know if this is feasible with viper?